config.json

```diff
@@ -517,6 +523,12 @@
             "ContainerName": "default"
           }
         },
+        "nodeReuse":{
```

Comment:
I'm not sure how dotnet/buildtools#864 connects to this. While your change is slightly better I would really prefer that be the default and not need to be passed in all the scripts.